format-objc-hook

```diff
@@ -19,18 +19,24 @@ objc_files=$(objc_files_to_format "$1")
 function format_objc() {
   success=0
   for file in $objc_files; do
-	difference=$("$DIR"/format-objc-file-dry-run.sh "$file" | diff "$file" - | wc -l)
+    difference=$("$DIR"/format-objc-file-dry-run.sh "$file" | diff "$file" - | wc -l)
 	
     if [ $difference -gt 0 ]; then
+        if [ $success -eq 0 ]; then
+            echo -e "🚸 Format and stage individual files:"
+        fi
     	# This is what the dev can run to fixup an individual file
-    	echo "\"$DIR\"/format-objc-file.sh '$file' && git add '$file';"
+    	echo -e "\"$DIR\"/format-objc-file.sh '$file' && git add '$file';"
     	success=1
```

Comment:
Not necessary, pre-existing issue - Having `success=1` mean that it is failing seems counter-intuitive to me, but I understand the exit code should be non-zero.